slack_sdk/webhook/async_client.py

```diff
@@ -33,6 +41,7 @@ def __init__(
         default_headers: Optional[Dict[str, str]] = None,
         user_agent_prefix: Optional[str] = None,
         user_agent_suffix: Optional[str] = None,
+        logger: Optional[logging.Logger] = None,
```

Comment:
Not to break backward compatibility, I intentionally added this arg as the last one in the arg list.